tests/functional/spec/save-come-back.spec.js

```diff
@@ -0,0 +1,118 @@
+import assert from 'assert'
+import chai from 'chai'
+import {getRandomString, startQuestionnaire, openQuestionnaire} from '../helpers'
+import devPage from '../pages/dev.page'
+import landingPage from '../pages/landing.page'
+import thankYou from '../pages/thank-you.page'
+import signOut from '../pages/sign-out.page'
+import reportingPeriod from '../pages/surveys/rsi/0102/reporting-period.page'
+import retailTurnoverPage from '../pages/surveys/rsi/0102/retail-turnover.page'
+import internetSalesPage from '../pages/surveys/rsi/0102/internet-sales.page'
+import changeInRetailTurnover from '../pages/surveys/rsi/0102/changes-in-retail-turnover.page'
+import SummaryPage from '../pages/summary.page'
+
+const expect = chai.expect
+
+describe('RSI - Save and restore test', function() {
+
+  it('Given I am completing survey 0102, when I select save and come back later, then I am redirected to sign out page, showing a message that my survey has been saved and I cant continue without re-authenticating', function() {
```

Comment:
Kept it as one to avoid duplication, but made it a lot shorter and easier to read